src/metadata_tool/tools/github_reports.clj

```diff
@@ -0,0 +1,216 @@
+(ns metadata-tool.tools.github_reports
+  (:require [clojure.string                 :as str]
+            [clojure.data.json              :as json]
+            [clojure.set                    :as set]
+            [clojure.walk                   :as wlk]
+            [clojure.java.io                :as io]
+            [clojure.pprint                 :as pp]
+            [metadata-tool.sources.github   :as gh]
+            [metadata-tool.sources.metadata :as md]))
+
+(def notice-finos-terms "This product includes software developed at the Fintech Open Source Foundation - https://www.finos.org")
+
+(def finos-github-orgs ["finos" "finos-plexus" "HadoukenIO" "symphonyoss"])
+
+; TODO - load this from metadata-tool
+(def finos-states ["active" "incubating" "released" "archived"])
+
+(def contributing-header "# contributing")
```

Comment:
the matcher later on is case sensitive, so `# CONTRIBUTING` would fail.

Reply:
I see that `readme-content` is defined as `(str/lower-case readme)`, so we should be fine.